cimr/defaults.py

```diff
@@ -1,5 +1,9 @@
 """Set of default values used in cimr"""
 
+#ANNOTURL = 'https://raw.githubusercontent.com/greenelab/cimr/master/cimr/data/annotation/'
+ANNOTURL = 'https://raw.githubusercontent.com/greenelab/cimr/crossmap/cimr/data/annotation/'
```

Comment:
This is temporary. Once the PR is approved, I will change it to the commented version (master).